### DISCUSSION TITLE
RVC Files Placement?

### DISCUSSION THREAD

**Discussion by QuietSkulls** [erew123/alltalk_tts#279] [Q&A]
Where do you place your trained RVC models in the Alltalk directory? (Clarification: Once I'm finished training my own RVC model; which files go where within the Alltalk tts directory?)

**Comment by erew123** [comment]
Hi @NerdistD 

Once you have enabled RVC in the Global settings menu, all the folders are created and base RVC models are downloaded automatically:

![image](https://github.com/user-attachments/assets/70c3ca8c-9b2a-4c51-9268-1a475c0cfd42)

As for the voice location, this is covered in the RVC documentation under "Voice Model Files"

![image](https://github.com/user-attachments/assets/f4e0bce7-3689-4b52-82c6-5427d46f3b6f)

Thanks

**Reply by QuietSkulls** [reply]
Thank you for your help!
